src/app/_components/home/activity/activity-home.css

```
@@ -973,6 +973,34 @@
   font-variant-numeric: tabular-nums;
 }
 
+/* Per-row workspace badge, only rendered by the aggregated cross-workspace
+   feed at /activity. Sits inline in the meta row beside the timestamp. */
+.wsa-feed__meta {
```

Stylelint (17.12.0)

[error] Expected class selector ".wsa-feed__meta" to be kebab-case (selector-class-pattern)

(selector-class-pattern)